src/shared/components/carousel/index.js

```diff
@@ -16,53 +16,74 @@ const settings = {
   slidesToShow: 1,
   slidesToScroll: 1,
   prevArrow: <Arrow direction="left"/>,
-  nextArrow: <Arrow direction="right"/>
+  nextArrow: <Arrow direction="right"/>,
+  responsive: [{
+    breakpoint: 769,
+    settings: {
+      centerPadding: '20%'
+    }
+  }, {
+    breakpoint: 481,
+    settings: {
+      centerPadding: '12%'
+    }
+  }]
 }
 
-const Carousel = ({ itemsList, modifier, size, onVideoClick, activeIndex, translationsList, isMobile }) => {
-  let items
-  const numberOfSlidesToShow = isMobile ? 1 : size
-  const finalSettings = { ...settings, slidesToShow: numberOfSlidesToShow }
+class Carousel extends Component {
+  render () {
+    const { itemsList, modifier, size, onVideoClick, activeIndex, translationsList, isMobile } = this.props
+    let items
```

Comment:
Put all the `const` first, then the `let`. Sorry for the nitpick, it's my OCD hehe.